src/auto-complete-element.js

```diff
@@ -16,6 +16,7 @@ export default class AutocompleteElement extends HTMLElement {
     const input = this.querySelector('input')
     const results = document.getElementById(owns)
     if (!(input instanceof HTMLInputElement) || !results) return
+    input.setAttribute('aria-owns', owns)
```

Comment:
`combobox-nav` requires `aria-owns` relationship to be on input & list since it is unaware of the parent element. This copies `auto-complete[aria-owns]` onto `input`.